lib/streamlit/elements/widgets/chat.py

```diff
@@ -316,10 +316,10 @@ def chat_input(
         # Use bottom position if chat input is within the main container
         # either directly or within a vertical container. If it has any
         # other container types as parents, we use inline position.
-        parent_block_types = set(self.dg._active_dg._parent_block_types)
+        anscestor_block_types = set(self.dg._active_dg._anscestor_block_types)
```

Comment:
` anscestor` is probably a typo or? This is probably supposed to be `ancestor`?

Reply:
oops -- I must have managed to misspell it throughout the PR 🙈

will fix